cpp/include/cudf/null_mask.hpp

```diff
@@ -202,30 +202,30 @@ rmm::device_buffer copy_bitmask(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /**
- * @brief Returns a bitwise AND of the bitmasks of columns of a table
+ * @brief Returns a pair of bitwise AND of the bitmasks of columns of a table and count of null bits
  *
  * If any of the columns isn't nullable, it is considered all valid.
  * If no column in the table is nullable, an empty bitmask is returned.
  *
  * @param view The table of columns
  * @param mr Device memory resource used to allocate the returned device_buffer
- * @return rmm::device_buffer Output bitmask
+ * @return A pair of resulting bitmask and count of null bits
  */
-rmm::device_buffer bitmask_and(
+std::pair<rmm::device_buffer, size_type> bitmask_and(
   table_view const& view,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /**
- * @brief Returns a bitwise OR of the bitmasks of columns of a table
+ * @brief Returns a pair of bitwise OR of the bitmasks of columns of a table and count of null bits
  *
  * If any of the columns isn't nullable, it is considered all valid.
  * If no column in the table is nullable, an empty bitmask is returned.
  *
  * @param view The table of columns
  * @param mr Device memory resource used to allocate the returned device_buffer
- * @return rmm::device_buffer Output bitmask
+ * @return A pair of resulting bitmask and count of null bits
```

Comment:
So the internal `bitmask_and` function returns the count of set bits. Where does the transformation happen to turn it into a count of unset bits?

Reply:
The main reason is that all current use cases require a count of unset bits in postprocessing. And now when I look back, I have to admit that returning the count of unset bits instead of set bits is a bit awkward.